README.md

```diff
@@ -41,8 +41,8 @@ released version of Element:
 1. Untar the tarball on your web server
 1. Move (or symlink) the `element-x.x.x` directory to an appropriate name
 1. Configure the correct caching headers in your webserver (see below)
-1. Configure the app by copying `config.sample.json` to `config.json` and
-   modifying it. See the [configuration docs](docs/config.md) for details.
+1. Configure the app by making a copy of the `config.sample.json` and renaming it as
+  `config.json` within the same directory. You can also customize it. See the [configuration docs](docs/config.md) for details.
```

Comment:
please match the column limit for the existing file as best as possible throughout.